spaceship/lib/spaceship/client.rb

```diff
@@ -630,6 +630,22 @@ def fetch_program_license_agreement_messages
       return all_messages
     end
 
+    # Get provider news from App Store Connect's "olympus" endpoint
+    def fetch_provider_news_messages
+      all_messages = []
+
+      messages_request = request(:get, "https://appstoreconnect.apple.com/olympus/v1/providerNews")
```

Comment:
Maybe it is better to leave it like this to stay consistent with the current code of Fastlane, but the result of the `request()` function should be a `messages_response`, not `messages_request`, no? 😄